tbx/static_src/sass/config/_variables.scss

```diff
@@ -31,6 +32,8 @@ $color--dark-grey: #1d2325;
     --color--accent-three: #{$color--coral-light};
     --color--accent-four: #{$color--purple};
     --color--accent-five: #{$color--white};
+    --color--link-icon: #{$color--grey-50};
+    --color--page: #{$color--dark-grey};
```

Comment:
Is this used for the text colour on the page? I think there's a `--colour-text` if so. I wonder if we should document what all the CSS custom properties are used for? Might be tricky for some of them (accents).

Reply:
It's a different colour to the text, and it was called 'page colour' in the designs, but maybe not the best name for it.

Reply:
Thanks to Nick I've got a better idea of where it is used. It's used for the arrows, and for the underline on some links e.g. in the header nav, the footer links and in the in page nav (last one may not be used). Would `--color--decoration` make sense?

Reply:
That sounds good!

Reply:
Fixed in 9cb0c859d8a28c934996d24cc4897cdcd7559fd3. It turned out that `--color-link-icon` was the one I had asked Nick about - so this is now called `--color--decoration`. I have removed `--color-page` as it was the same as `--color--background` - and makes sense that the tag uses the inverse of the background colour when hovered.